Makefile

```diff
@@ -1,27 +1,30 @@
-CC = gcc
-LINKER = gcc
+LUA         = 5.1
+LUA_CFLAGS  = $(shell pkg-config --cflags lua$(LUA))
```

Comment:
Hi, in our linux dist the `lua.pc` is located at `/usr/lib64/pkgconfig/lua.pc`.  I would like to keep our Makefile as it is.. If you could revert the Makefile change it would help :)

Thank you, appreciate it!

Reply:
Hello, if the lua.pc is located in /usr/lib64/pkgconfig/lua.pc I don't think there is a problem since pkg-config itself is in charge of looking for the .pc paths, it already looks for them automatically when using pkg-config --cflags lua$(LUA).

I don't think there are any errors, unless pkgconfig has a problem and is not recognizing the routes set.